applications/FluidDynamicsApplication/custom_strategies/strategies/residualbased_simple_steady_scheme.h

```diff
@@ -140,7 +142,7 @@ class ResidualBasedSimpleSteadyScheme : public Scheme<TSparseSpace, TDenseSpace>
     if (SteadyLHS.size1() != 0)
       noalias(LHS_Contribution) += SteadyLHS;
 
-    AddRelaxation(rCurrentElement->GetGeometry(), LHS_Contribution, RHS_Contribution, CurrentProcessInfo);
+    // AddRelaxation(rCurrentElement->GetGeometry(), LHS_Contribution, RHS_Contribution, CurrentProcessInfo);
```

Comment:
please remove this, leaving it commented can be confusing

Reply:
done